eng/pipelines/templates/stages/archetype-conda-release.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  CondaArtifacts: []
+  TestPipeline: false
+  DependsOn: Build
+  ArtifactName: 'not-specified'
+  ServiceDirectory: 'not-specified'
+
+
+stages:
+  - ${{if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
+    - ${{ each artifact in parameters.CondaArtifacts }}:
+      - stage: Release_${{artifact.name}}
+        displayName: 'Release: ${{artifact.name}}'
+        dependsOn: ${{parameters.DependsOn}}
+        condition: and(succeeded(), ne(variables['SetDevVersion'], 'true'), ne(variables['Skip.Release'], 'true'), ne(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-python-pr'))
+        jobs:
+          - deployment: CondaRelease
+            displayName: "Publish Conda Artifacts"
+            condition: ne(variables['Skip.TagRepository'], 'true')
+            environment: pypi
+
+            pool:
+              name: azsdk-pool-mms-ubuntu-1804-general
+              vmImage: MMSUbuntu18.04
+
+            strategy:
+              runOnce:
+                deploy:
+                  steps:
+                    - checkout: self
+                    - pwsh: |
+                        Get-ChildItem -Recurse $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}
+                      workingDirectory: $(Pipeline.Workspace)
+                      displayName: Output Visible Conda Artifacts
```

Comment:
qq, what does this do? Is it there as a debugging tool to see all the files or is it necessary? (Just curious, I litter yml files with Get-ChildItem -Recurse for debugging)

Reply:
I assume this is just a placeholder for the time being until we figure out the necessary steps to publish.

Reply:
This is exactly correct.